### DISCUSSION TITLE
Can you override the findSlug endpoint?

### DISCUSSION THREAD

**Discussion by michaelnapl** [ComfortablyCoding/strapi-plugin-slugify#114] [Q&A]
Any thoughts on being able to override the findSlug endpoint (what's returned)?

I have a specific application where I actually need to return data in a defined format.

I know you can override the default find/findOne methods, but not sure if or how you could do it for the findSlug one?

Possible? Or impossible?

Thanks

**Comment by ComfortablyCoding** [comment]
I am not sure myself, I will need to research this.
